tasks/webhook

```diff
@@ -146,7 +146,7 @@ class ReleaseHandler(github_handler.GithubHandler):
             return None
 
         body = {
-            'event': 'pull_request',
+            'event': event,
```

Comment:
Good call, and then the whole "build body and publish it to webhook" can be factorized.